sdks/java/core/src/test/java/org/apache/beam/sdk/io/WriteFilesTest.java

```diff
@@ -329,7 +329,11 @@ public void testWithRunnerDeterminedShardingUnbounded() throws IOException {
   }
 
   @Test
-  @Category({NeedsRunner.class, UsesUnboundedPCollections.class, UsesTestStream.class})
+  @Category({
+    NeedsRunner.class,
+    UsesUnboundedPCollections.class,
+    UsesTestStreamWithProcessingTime.class
```

Comment:
maybe keep the `UsesTestStream.class` category as well?

Reply:
Updated